ui/components/thing-counters.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default function ThingCounter({ count, singular, plural }, { router }) {
+  const query = router.location.query;
+  const queryKeys = Object.keys(query);
+  const noun = count === 1 ? singular : plural;
+  const verb = count === 1 ? 'matches' : 'match';
+  const classes = count === 0 ? 'alert p1 m1 border' : '';
+
+  const unfiltered = queryKeys.length === 0 || (queryKeys.length === 1 && queryKeys.indexOf('page') !== -1);
```

Comment:
This is a good, descriptive variable name. The logic feels a little hard to follow, so I'll throw out some odds and ends that may help in case you aren't familiar:
* `Object.assign({}, router.location.query)` would make a shallow copy of the query
* `del someObject.key` follows Python expectations for deleting an attribute. It's a noop if `key` isn't present
* `someArray.includes(key)` returns a bool and should be polyfilled for browser support already

Maybe we do something like:
```js
const queryWithoutPage = Object.assign({}, router.location.query);
del queryWithoutPage.page

const unfiltered = Object.keys(queryWithoutPage).length > 0
```
That doesn't feel particularly elegant, either...